src/members/models.py

```diff
@@ -10,19 +9,9 @@ class Member(AbstractUser):
     UUID = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     username = models.CharField(max_length=70, unique=True)
     email = models.CharField(max_length=320, unique=True, null=False, blank=False)
-    password = models.CharField(max_length=120,
-                                validators=[
-                                    RegexValidator(
-                                        regex='^(?=.*[a-z])(?=.*[A-Z])'
-                                              '(?=.*\\d)(?=.*[@$!%*?&])'
-                                              '[A-Za-z\\d@$!%*?&]{8,}$',
-                                        message='Enter password containing min 8 characters, '
-                                                'min 1 upper and lower case letter, '
-                                                '1 number and 1 special character')
-                                ], null=True, blank=True)
+    password = models.CharField(max_length=120, null=True, blank=True)
```

Comment:
Should the password field be nullable?

Reply:
It should not, good that you caught that!